keyboards/printedpad/info.json

```
@@ -0,0 +1,46 @@
+{
+    "manufacturer": "Prota Designs",
+    "keyboard_name": "Printed Pad",
+    "maintainer": "noahbei",
+    "bootloader": "stm32-dfu",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true,
+        "oled": true
+    },
+    "matrix_pins": {
+        "cols": ["B2", "B1", "B0"],
+        "rows": ["A10", "C9", "A8", "A9"]
+    },
+    "processor": "STM32F072",
+    "url": "",
+    "usb": {
+        "device_version": "1.0.0",
+        "pid": "0x0001",
+        "vid": "0x5044"
+    },
+    "layouts": {
+        "LAYOUT_ortho_4x3": {
```

This isn't really `ortho_4x3`
```suggestion
        "LAYOUT": {
```

And all of the keymaps would need to be updated too.